libv4v/src/interposer.c

```diff
@@ -265,6 +267,16 @@ INTERPOSE (connect, int, int sockfd, const struct sockaddr * addr,
   if (v4v_map_sa_to_v4va (&peer, addr, addrlen))
     return -EINVAL;
 
+  if (getenv ("V4V_BIND_CLIENT")) {
+    me.domain = V4V_DOMID_ANY;
+    me.port = peer.port + 10000;
```

Comment:
why peer.port + 10000?  perhaps let the client set it using the above env variable?